jquery.overlay.js

```diff
@@ -4,6 +4,11 @@
  * Repository: https://github.com/yuku-t/jquery-textoverlay
  * License:    MIT
  * Author:     Yuku Takahashi
```

Comment:
not exactly =/ what is protocol for that, should i add ", modified by Niki Castle" or something?